packages/surveys/src/exit_json.ts

```diff
@@ -0,0 +1,70 @@
+export const surveyJSON = {
+  pages: [
+    {
+      elements: [
+        {
+          description:
+            "We ask again just to check for typos during registration or accidental selection of a different child at the start of the study.",
+          inputType: "date",
+          isRequired: true,
```

Comment:
Not a big deal, but why did we change the exit JSON to also JSON?

Reply:
Not totally sure I understand the question. Are you asking why I changed the exit JSON to a JS object?
If so: the reason is that the jsPsych `survey` plugin now takes the `survey_json` as a JS object rather than JSON string, so I thought we could just store this as JS instead (since that gives us syntax highlighting, formatting etc. and is a little less annoying to work with). But we could stick with a JSON string and use JSON.parse() to pass it into the survey plugin, if that's preferable?